homework/App.js

```diff
@@ -12,18 +12,31 @@ class App {
    * @param {string} url The GitHub URL for obtaining the organization's repositories.
    */
   async initialize(url) {
-    // Add code here to initialize your app
-    // 1. Create the fixed HTML elements of your page
-    // 2. Make an initial XMLHttpRequest using Util.fetchJSON() to populate your <select> element
-
     const root = document.getElementById('root');
 
-    Util.createAndAppend('h1', root, { text: 'It works!' }); // TODO: replace with your own code
-
+    const repos = await Util.fetchJSON(url);
```

Comment:
Hi Ahmad - you may want to move this inside the `try` block as you wouldn't be able to catch any errors generated by `fetchJSON` method 🙂